src/nplinker/genomics/bigscape/runbigscape.py

```diff
@@ -15,22 +15,65 @@ def run_bigscape(
     antismash_path: str | PathLike,
     output_path: str | PathLike,
     extra_params: str,
-):
-    bigscape_py_path = "bigscape.py"
-    logger.info(
-        f'run_bigscape: input="{antismash_path}", output="{output_path}", extra_params={extra_params}"'
-    )
+    version: int = 1,
```

Comment:
Since `version` can only be `1` or `2`, we can use [Literal](https://docs.python.org/3.10/library/typing.html#typing.Literal) to annotate the type: `
```suggestion
    version: Literal[1, 2] = 1,
```